tests/test_anchor_box.py

```diff
@@ -0,0 +1,60 @@
+# Copyright (c) MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+
+import torch
+from parameterized import parameterized
+
+from monai.apps.detection.utils.anchor_utils import AnchorGenerator
+from monai.utils import optional_import
+from tests.utils import assert_allclose
+
+torch_anchor_utils, _ = optional_import("torchvision.models.detection.anchor_utils", version="0.12")
+image_list, _ = optional_import("torchvision.models.detection.image_list", version="0.12")
+
+
+TEST_CASES_2D = []
+TEST_CASES_2D.append(
+    [
+        {"sizes": ((10, 12, 14, 16), (20, 24, 28, 32)), "aspect_ratios": ((1.0, 0.5, 2.0), (1.0, 0.5, 2.0))},
+        (5, 3, 128, 128),
+        ((5, 7, 64, 32), (5, 7, 32, 16)),
+    ]
+)
+
+
+class TestAnchorGenerator(unittest.TestCase):
```

Comment:
could you please help add a torchscript test? like this one:
https://github.com/Project-MONAI/MONAI/blob/1516ca758090dd8ed2890b64554a77f681a0e224/tests/test_se_blocks.py#L68-L72